pkg/system/constants.go

```diff
@@ -8,6 +8,8 @@ const (
 	LEAccountSecretName  = "acorn-le-account"
 	DefaultUserNamespace = "acorn"
 	DNSSecretName        = "acorn-dns"
+	IngressName          = "acorn-ingress"
```

Comment:
maybe we can be a bit more explicit about the name of this and its purpose? Thinking `AcornDNSTestIngres`...a bit longer/wordier, but clearer in its intent?

Reply:
Sure! We can do something like that.